bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2220,7 +2248,7 @@ void ReplicaImp::tryToSendReqMissingDataMsg(SeqNum seqNumber, bool slowPathOnly,
   const bool missingBigRequests = (!missingPrePrepare) && (!seqNumInfo.hasPrePrepareMsg());
 
   ReplicaId firstRepId = 0;
```

Comment:
Confusing name firstRepId - reply? replica? report?

Reply:
I haven't touched this